app/services/case_filter/open_case_status_filter.rb

```diff
@@ -26,11 +26,24 @@ def available_choices
           end
         end
       end
+
+      offender_sar_move_state_filter_option_to_end(state_choices, "rejected")
+
       { filter_open_case_status: state_choices }
     end
 
   private
 
+    # For users dealing with Offender SARs
+    # To ensure 'rejected' is at the end of the filter list, .delete returns the
+    # value of the given key. It is then immediately assigned back.
+    def offender_sar_move_state_filter_option_to_end(filters, filter_key)
+      if @user.permitted_correspondence_types.any? { |h| h[:abbreviation] == "OFFENDER_SAR" }
```

Comment:
Reused the method as it now includes a conditional. Without the conditional non branston users had an unwanted filter option named `filter open case status rejected`

Reply:
I did that just in case it needed to be a new or different key in the future, as is easier to change. Happy to remove if needs be?

Reply:
That would be easy enough to change in the future anyway. Probably better to design this for how you need it to work now.